components/StyledInputAmount.js

```diff
@@ -19,7 +19,7 @@ const StyledInputAmount = ({ currency, min, max, value, onChange, ...props }) =>
       min={min}
       max={max}
       value={isNil(value) || value === '' ? '' : clamp(value, !min ? 0 : 1, max)}
-      prepend={getCurrencySymbol(currency)}
+      prepend={props.pre || getCurrencySymbol(currency)}
```

Comment:
The `pre` should be removed from the props (like `min`, `max`, `value`...etc) - we should not need pass it down to `StyledInputGroup`. We also need to add it to the `propTypes`.

Reply:
I added the prepend attribute in `CreateVirtualCardsForm` to override the one from `StyledInputAmount component` instead of passing down `pre`